addons/website_sale_delivery_mondialrelay/models/website.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class WebsiteMondialRelay(models.Model):
+    _inherit = 'website'
+
+    def _prepare_sale_order_values(self, partner, pricelist):
+        self.ensure_one()
+        values = super()._prepare_sale_order_values(partner, pricelist)
+        # never use Mondial Relay shpping address as default.
```

Comment:
while you modify other files, maybe just add the missing `i` here
```suggestion
        # never use Mondial Relay shipping address as default.
```